plugins/ssr/ssrcommon.ml

```diff
@@ -1436,7 +1436,8 @@ let tclINTRO ~id ~conclusion:k = Goal.enter begin fun gl ->
   let original_name = Rel.Declaration.get_name decl in
   let already_used = Tacmach.New.pf_ids_of_hyps gl in
   let id = match id, original_name with
-    | Some id, _ -> id
+    | `Id id, _ -> id
+    | `Seed id, _ -> mk_anon_id id already_used
     | _, Name id ->
```

Comment:
Using a defined ADT and an exhaustive match here will make the code more robust w.r.t. extensions.